metadata-ingestion/docs/sources/datahub/datahub_recipe.yml

```diff
@@ -0,0 +1,23 @@
+pipeline_name: datahub_source_1
+datahub_api:
+  server: "http://localhost:8080"  # Migrate data from DataHub instance on localhost:8080
+  token: "<token>"
+source:
+  type: datahub
+  config:
+    include_all_versions: false
+    database_connection:
+      scheme: "mysql+pymysql"  # or "postgresql+psycopg2" for Postgres
+      host_port: "<database_host>:<database_port>"
+      username: "<username>"
+      password: "<password>"
+      database: "<database>"
+    kafka_connection:
+      bootstrap: "<boostrap_url>:9092"
+      schema_registry_url: "<schema_registry_url>:8081"
+    stateful_ingestion:
+      enabled: true
+      ignore_old_state: false
+  extractor_config:
+    set_system_metadata: false  # Replicate system metadata
```

Comment:
eventually I'd like to move these to the "flags" section that we added

Reply:
Ah I thought the flags section would be for relatively temporary flags. You're thinking we'd store permanent configs in there as well?

Reply:
let's talk about it more later - we do need a home for some of these things, but not sure where that should be